src/Templating/Twig.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Framework3\Templating;
+
+use Framework3\ClassFinder\AutoloadTools;
+use Framework3\Config\ConfigBag;
+use Framework3\Exception\AppException;
+use Twig\Environment;
+use Twig\Loader\FilesystemLoader;
+
+/**
+ * Class Twig
+ *
+ * @package Climb\Templating
+ */
+class Twig
+{
+    /**
+     * @var AutoloadTools
+     */
+    private AutoloadTools $autoloadTools;
+
+    /**
+     * @var ConfigBag
+     */
+    private ConfigBag $config;
+
+    /**
+     * @var string
+     */
+    private string $baseDir;
+
+    /**
+     * @param AutoloadTools $autoloadTools
+     * @param ConfigBag     $config
+     * @param string        $baseDir
+     */
+    public function __construct(AutoloadTools $autoloadTools, ConfigBag $config, string $baseDir)
+    {
+        $this->autoloadTools = $autoloadTools;
+        $this->config = $config;
```

Comment:
Codacy found an issue: [Equals sign not aligned with surrounding assignments; expected 8 spaces but found 1 space](https://app.codacy.com/gh/pierregaimard/framework3/pullRequest?prid=6279691)